site/app/libraries/database/DatabaseQueries.php

```diff
@@ -600,7 +600,14 @@ public function loadThreadBlock($categories_ids, $thread_status, $unread_threads
         }
         $query_offset = ($blockNumber - 1) * $blockSize;
         $this->buildLoadThreadQuery($categories_ids, $thread_status, $unread_threads, $show_deleted, $show_merged_thread, $current_user, $query_select, $query_join, $query_where, $query_order, $query_parameters, true, true);
-        $query = "SELECT {$query_select} FROM threads t {$query_join} WHERE {$query_where} ORDER BY {$query_order} LIMIT ? OFFSET ?";
+        if ($sortOption == "time") {
+            $query_order = "t.id ASC";
+        }
+        elseif ($sortOption == "reverse-time") {
+            $query_order = "t.id DESC";
+        }
```

Comment:
Do we need an "else" case here?

Reply:
So, by default, it was not set here. When I checked value of $query_order was  set by buildLoadThreadQuery as "row_number". So I did not write else condition not to delete the behavior that works without sorting. What would be a better approach?